files/en-us/mdn/contribute/changelog/index.html

```diff
@@ -12,11 +12,29 @@
 
 <h2 id="july_2021">July 2021</h2>
 
-<h3 id="heading_markup_for_note_and_warning_boxes">Heading markup for note and warning boxes</h3>
-
-<p><a href="/en-US/docs/MDN/Guidelines/CSS_style_guide#.note.notecard">Note</a> and <a href="/en-US/docs/MDN/Guidelines/CSS_style_guide#.notecard.warning">warning</a> boxes no longer have a separate <code>&lt;h4&gt;</code> heading for the title (e.g. <code>&lt;h4&gt;Warning&lt;/h4&gt;</code> ). Instead, the the first paragraph begins with title text, as shown: <code>&lt;strong&gt;Note:&lt;/strong&gt; </code> and <code>&lt;strong&gt;Warning:&lt;/strong&gt; </code>. Note that a space character is required after the closing <code>&lt;/strong&gt;</code> tag/before the content).</p>
-
-<p>See our <a href="/en-US/docs/MDN/Guidelines/CSS_style_guide#.note.notecard">Guide to classes and styles used in MDN content</a> guide for further information and syntax guides.</p>
+<h3>Updates to CSS style guide for Markdown</h3>
+
+<p>Multiple updates to the <a href="/en-US/docs/MDN/Guidelines/CSS_style_guide">CSS style guide</a> to reflect the move towards Markdown, and encourage authors to write HTML in a Markdown-compatible way.</p>
+
+<ul>
+  <li>
+    <p><a href="/en-US/docs/MDN/Guidelines/CSS_style_guide#.note.notecard">Note</a> and <a href="/en-US/docs/MDN/Guidelines/CSS_style_guide#.notecard.warning">warning</a> boxes no longer have a separate <code>&lt;h4&gt;</code> heading for the title (e.g. <code>&lt;h4&gt;Warning&lt;/h4&gt;</code> ). Instead, the the first paragraph begins with title text, as shown: <code>&lt;strong&gt;Note:&lt;/strong&gt; </code> and <code>&lt;strong&gt;Warning:&lt;/strong&gt; </code>. Note that a space character is required after the closing <code>&lt;/strong&gt;</code> tag/before the content).</p>
```

Comment:
A duplicate `the`:
```suggestion
    <p><a href="/en-US/docs/MDN/Guidelines/CSS_style_guide#.note.notecard">Note</a> and <a href="/en-US/docs/MDN/Guidelines/CSS_style_guide#.notecard.warning">warning</a> boxes no longer have a separate <code>&lt;h4&gt;</code> heading for the title (e.g. <code>&lt;h4&gt;Warning&lt;/h4&gt;</code> ). Instead, the first paragraph begins with title text, as shown: <code>&lt;strong&gt;Note:&lt;/strong&gt; </code> and <code>&lt;strong&gt;Warning:&lt;/strong&gt; </code>. Note that a space character is required after the closing <code>&lt;/strong&gt;</code> tag/before the content).</p>
```
(Not me making the approval, though)

Reply:
thanks!